packages/governance/src/contractGovernorKit.js

```diff
@@ -1,16 +1,57 @@
 import { Fail } from '@agoric/assert';
-import { makeTracer } from '@agoric/internal';
-import { prepareExoClassKit } from '@agoric/vat-data';
+import { UnguardedHelperI, makeTracer } from '@agoric/internal';
+import { M, prepareExoClassKit } from '@agoric/vat-data';
 import { E } from '@endo/eventual-send';
+import {
+  InvitationShape,
+  InstanceHandleShape,
+  InstallationShape,
+} from '@agoric/zoe/src/typeGuards.js';
+import { TimestampShape } from '@agoric/time';
 import { setupApiGovernance } from './contractGovernance/governApi.js';
 import { setupFilterGovernance } from './contractGovernance/governFilter.js';
 import {
   CONTRACT_ELECTORATE,
   setupParamGovernance,
 } from './contractGovernance/governParam.js';
+import { ClosingRuleShape, ParamChangesSpecShape } from './typeGuards.js';
 
 const trace = makeTracer('CGK', false);
 
+const ContractGovernorKitI = {
+  helper: UnguardedHelperI,
```

Comment:
Why not expand `HelperI`?

Reply:
1) because I have other work to do
2) helpers aren't as important to guard. they're called only from internal code.

Reply:
Guards protect against bugs and accidents as well as malice and intentional attacks. 
It doesn't seem like much work or much maintenance burden to include guards for `helper`.  I see that `UnguardedHelper` makes it possible to not include those details. Why isn't it worth it to include them?

Reply:
> It doesn't seem like much work

I disagree but if you think it's negligible please go ahead

Reply:
> Why isn't it worth it to include them?

Because helpers are only called from within the Exo and any type errors would be detected by TypeScript, which is more expressive. The runtime guards are very important for methods that can be called remotely, but for local methods they impose an upfront cost of defining them and a recurring cost of changing them whenever the internal API is refactored. It's not worth the work.